webserver/__init__.py

```diff
@@ -31,6 +33,9 @@ def create_app():
     from kafka_connection import init_kafka_connection
     init_kafka_connection(app.config['KAFKA_CONNECT'])
 
+    # Redis connection
+    app.redis = Redis()
```

Comment:
the specific problem was with postgres connections being shared between threads, which wasn't good. We can use a `Redis` in the whole app with no problem. We might want to use a pattern like @gentlecat suggests, like https://github.com/metabrainz/acousticbrainz-server/blob/6d0099184f00843b3f9c13d94e27803e4d5a28dc/db/__init__.py#L16-L22